src/index.html

```diff
@@ -9,10 +13,40 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="../src/styles/blocks/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card"
+     data-qa="card">
+
+      <div class="card__photo"></div>
+      <div class="card__info">
+
+        <h2 class="card__title">APPLE A1419 iMac 27" Retina<br>
+          5K Monoblock (MNED2UA/A)
```

Comment:
Using a `<br>` tag within a heading element like `<h2>` is generally not recommended as it can affect accessibility and semantic HTML structure. Consider using CSS for styling and spacing instead.